src/Injector.js

```diff
@@ -1,107 +1,79 @@
 import { Component, PropTypes } from 'react';
-import values from 'lodash/object/values';
+import identity from 'lodash/utility/identity';
 import mapValues from 'lodash/object/mapValues';
-import invariant from 'invariant';
-import isPlainObject from 'lodash/lang/isPlainObject';
-import isFunction from 'lodash/lang/isFunction';
+import shallowEqual from './utils/shallowEqual';
 
-export default class ReduxInjector extends Component {
+export default class Injector extends Component {
   static contextTypes = {
     redux: PropTypes.object.isRequired
   };
 
   static propTypes = {
     children: PropTypes.func.isRequired,
+    select: PropTypes.func.isRequired,
```

Comment:
I thought about that.. But then, people are going to want arrays of string keys for keyPaths and we're back to cursors :-). I think functions (or lambdas) work great here.